--- origsrc/Maze-22/olpcgames/activity.py	2012-04-20 11:05:44.000000000 -0500
+++ src/Maze-22/olpcgames/activity.py	2013-01-28 19:52:44.758723400 -0600
@@ -169,7 +169,6 @@ class PygameActivity(activity.Activity):
             self._pgc.connect_game(self.game_handler or self.game_name)
             # XXX Bad coder, do not hide in a widely subclassed operation
             # map signal does not appear to show up on socket instances
-            gtk.gdk.threads_init()
             return self._pgc
         else:
             import hippo
